hail/python/hailtop/batch/__init__.py

```diff
@@ -1,15 +1,17 @@
 import nest_asyncio  # type: ignore
 
-from .batch import Batch
+from .batch import Batch, Resource
 from .batch_pool_executor import BatchPoolExecutor
 from .backend import LocalBackend, ServiceBackend
 from .utils import BatchException
 
 __all__ = ['Batch',
+           'Resource',
```

Comment:
I prefer we do not directly expose `Resource` to our users. Moreover, the fact `Resource` is in the scope of `.batch` is an implementation detail. It properly resides `resource`.

Instead, let's reference it in the tests as `from hailtop.batch.resource import Resource`.